src/button/tests/functional/Button.ts

```diff
@@ -0,0 +1,73 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=button')
+		.setFindTimeout(5000);
+}
+
+registerSuite({
+	name: 'Button',
+
+	'button should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('button:first-of-type')
```

Comment:
I think id is not always bad. In the example page, each example is unique. There will always be one `example-1` on the page. And I am not trying to consistently style multiple examples, but access each of them individually.